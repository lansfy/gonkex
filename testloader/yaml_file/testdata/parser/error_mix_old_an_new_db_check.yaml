- name: WHEN use deprecated and new style with dbChecks loader MUST fail with error
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  dbQuery: "SELECT ALL"
  dbResponse:
    - item1
  dbChecks:
    - dbQuery: "UPDATE ALL"
      dbResponse:
        - item2
