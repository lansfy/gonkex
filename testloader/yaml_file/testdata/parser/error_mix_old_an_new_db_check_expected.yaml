- Error: "process 'testdata/parser/error_mix_old_an_new_db_check.yaml': mix of old dbQuery/dbResponse and dbChecks in one test forbidden"
